benches/common/macros.rs

```diff
@@ -34,34 +38,14 @@ macro_rules! bench_binop_fn(
     ($name: ident, $t1: ty, $t2: ty, $binop: path) => {
         fn $name(bh: &mut criterion::Criterion) {
             use rand::SeedableRng;
+            use std::hint::black_box;
+
             let mut rng = IsaacRng::seed_from_u64(0);
             let a = rng.random::<$t1>();
             let b = rng.random::<$t2>();
 
             bh.bench_function(stringify!($name), move |bh| bh.iter(|| {
-                $binop(&a, &b)
```

Comment:
Hey, sorry if I'm being a bit slow here. Why is this gone? Was this just dead code?